src/style/_layout.scss

```diff
@@ -207,31 +205,29 @@
 
         h4
         {
-            font-size: 14px;
 
             flex: 1;
 
             margin: 0;
 
-            @include text_headline();
+            @include text_title($size: $opblock-section-header-h4-font-size);
         }
     }
 
     .opblock-summary-method
     {
-        font-size: 14px;
-        font-weight: bold;
+        display: flex;
```

Comment:
Is this left over from my PR?
If so you should probable delete the flex props and re-add `text-align: center`

Reply:
done. cheers!